imports/ui/components/study_plan/Tab.less

```diff
@@ -0,0 +1,50 @@
+// ========================================
```

Comment:
Awesome abstraction of styles from dumb components! We didn't think we'd need this at the beginning of our project but now, looking at this CSS stylesheet, it does indeed help a lot in maintaining CSS styles in the future! Next time, we can even use a React style selector that packages these CSS files into an object and we can import it into the React components that need that stylesheet, making it even more maintainable!